app/controllers/alert_controller.rb

```diff
@@ -0,0 +1,11 @@
+class AlertController < ApplicationController
+  def create
+    email = Email.create!(
+      address: params.require(:recipient_address),
```

Comment:
```suggestion
      address: params.require(:address),
```

Small thing, I'm not sure if `recipient` adds anything here although I don't really mind.

Reply:
Yeah fair point, i'll remove it.